tests/std/tests/P0009R18_mdspan_layout_right/test.cpp

```diff
@@ -51,15 +51,15 @@ constexpr void check_members(const extents<IndexType, Extents...>& ext, index_se
     using Ext2           = extents<OtherIndexType, Extents...>;
     using Mapping2       = layout_right::mapping<Ext2>;
 
-#pragma warning(push) // TRANSITION, "/analyze:only" BUG?
-#pragma warning(disable : 28020) // The expression '0<=_Param_(1)&&_Param_(1)<=1-1' is not true at this call
+#ifndef __clang__ // FIXME, Clang suddenly cannot digest this
     { // Check construction from other layout_right::mapping
         Mapping m1{ext};
         Mapping2 m2{m1};
         assert(m1 == m2);
         static_assert(is_nothrow_constructible_v<Mapping2, Mapping>);
         // Other tests are defined in 'check_construction_from_other_right_mapping' function
     }
+#endif

```

Comment:
Suggestion for followup: This `#endif` should be commented. (The `#ifndef` is FIXME so it'll be resolved before the final merge, but if we need to convert it into a workaround with a cited LLVM bug, commenting the `#endif` will avoid forgetting `// __clang__` later.)